bundle/src/main/java/com/adobe/acs/commons/adobeio/core/service/impl/HttpClientServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2013 - 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.adobeio.core.service.impl;
+
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.osgi.service.component.annotations.Component;
+
+import com.adobe.acs.commons.adobeio.core.service.HttpClientService;
+
+@Component(service = {HttpClientService.class},
+immediate = true)
+public class HttpClientServiceImpl implements HttpClientService {
+
+    @Override
+    public CloseableHttpClient getHttpClient() {
+        return HttpClientBuilder.create().build();
```

Comment:
missing timeouts and per, my comment on the service interface, i'm not sure why a service is necessary here at all.